q2_feature_table/tests/test_subsample.py

```diff
@@ -12,7 +12,7 @@
 import numpy.testing as npt
 from biom.table import Table
 
-from q2_feature_table import subsample
+from q2_feature_table import subsample_ids
 
 
 class SubsampleTests(TestCase):
```

Comment:
```suggestion
class SubsampleIDsTests(TestCase):
```
I think you had the right idea when you changed the name of this class @hagenjp - it's consistent with the name changes you've implemented within all of the subsequent tests, so changing the class name as well makes sense to me!